crates/uv/tests/it/pip_install_scenarios.rs

```diff
@@ -3378,18 +3378,24 @@ fn transitive_prerelease_and_stable_dependency_many_versions_holes() {
       × No solution found when resolving dependencies:
       ╰─▶ Because only the following versions of package-c are available:
               package-c<1.0.0
-              package-c>=2.0.0a5,<=2.0.0a7
+              package-c==2.0.0a5
+              package-c==2.0.0a6
+              package-c==2.0.0a7
```

Comment:
Honestly no clue what's up here?

Reply:
I think I need to gate this logic to avoid applying if any of the ranges include pre-releases?

Reply:
Fixed.